homeassistant/components/vicare/binary_sensor.py

```diff
@@ -198,7 +209,7 @@ def __init__(
         self._device_config = device_config
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, device_config.getConfig().serial)},
-            name=device_config.getModel(),
+            name=f"{device_config.getModel()}-{device_config.getConfig().serial}",
```

Comment:
Does Vicare support a name or something better as the serial for naming?

Before this change, the entity was for example `ViCare Circulation pump active`
After it would be `TEST-123456789 Circulation pump active` which is not really userfriendly.

Reply:
In the App it looks like this:

![image](https://github.com/home-assistant/core/assets/9592452/bcc4fa93-ffa7-4a85-8454-8eba827b6191)

I have registered both comm modules to one site. The site has a name I could choose on setup I think. The device (heater) has a predefined name (Kessel) which is not changeable. 

No idea how this is reflected in the api.

Reply:
I'm not familiar with ViCare but how many devices has a user usually?

If the most only have one device, I would suggest to add the serial only if two or more devices are returned by the api

Reply:
Good idea. For most users, they probably have only one device.